docs/contribute/new-provider.rst

```diff
@@ -8,8 +8,8 @@ The actual implementation gives two providers: :ref:`grid5000` and
 then implementing a new one is easy as 500 lines of Python code.
 
```

Comment:
Actually, I fixed it in the next #147. But during my process of splitting #147 in two, I cannot manage to edit the diff file to remove the reference to the static provider. So I decided to leave this paragraph has it is, with the knowledge that a fix is coming in #147.